internal/serverinstall/k8s.go

```diff
@@ -721,6 +721,13 @@ func (i *K8sInstaller) Uninstall(ctx context.Context, opts *InstallOpts) error {
 			LabelSelector: "app=" + runnerName,
 		},
 	)
+	if err != nil {
+		ui.Output(
+			"Error watching deployments: %s", clierrors.Humanize(err),
```

Comment:
This would actually just be an error initializing the watch variable. It doesn't watch the deployment until a bit further down

```suggestion
			"Error creating deployments watcher: %s", clierrors.Humanize(err),
```

Reply:
Fixed.